ads/inabox/inabox-messaging-host.js

```diff
@@ -126,10 +126,21 @@ export class InaboxMessagingHost {
    * @return {boolean}
    */
   handleSendPositions_(iframe, request, source, origin) {
+    const viewportRect = this.positionObserver_.getViewportRect();
+    const targetRect =
+        layoutRectFromDomRect(iframe./*OK*/getBoundingClientRect());
```

Comment:
we decided not to. This is non AMP host. To do it in a vsync, I'll have to first polyfill `#requestAnimationFrame` first. Inabox should be responsible to request position only once in single animationframe.